src/nodes/QskBoxRendererEllipse.cpp

```diff
@@ -859,17 +990,24 @@ static inline void qskRenderBorder( const QskBoxRenderer::Metrics& metrics,
 
     if ( colors.isMonochrome() )
     {
-        qskRenderBorderLines( metrics, orientation, line, BorderMapSolid( c.rgb( Qsk::Left ) ) );
+        qskRenderBorderLines( metrics, orientation, line, BorderMapSolid( c.gradient( Qsk::Left ).startColor().rgb() ) );
```

Comment:
rgb -> rgba

To avoid running into this trap I would introduce something like this at the top of this file:

static inline QRgb qskRgbGradient( const QskGradient& gradient )
{
    return gradient.startColor().rgba();
}

static inline QRgb qskRgbBorder( const QskBoxBorderColors& borderColors )
{
    return qskRgbGradient( borderColors.gradient( Qsk::Left ) );
}

Reply:
I introduced those methods above and one qskRgbGradientEnd() for the RGBA of the end color, since we need this function as well.